msal/src/main/java/com/microsoft/identity/client/IAuthenticationResult.java

```diff
@@ -25,14 +25,26 @@
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
 
+import com.microsoft.identity.common.internal.result.IBaseAuthenticationResult;
+
 import java.util.Date;
 
-public interface IAuthenticationResult {
+public interface IAuthenticationResult extends IBaseAuthenticationResult {
+
+
+    /**
+     * Gets the Account.
+     *
+     * @return The Account to get.
+     */
+    @NonNull
+    IAccount getAccount();
```

Comment:
If this interface inheritance structure stays, we can remove these `@Override` annotations and the re-declaration.

I'm still pondering this interface design....

Reply:
At a minimum, we should remove the javadoc so that it doesn't need to be updated in two places. It will be automatically surfaced by any decent editor from the interface superclass anyway

Reply:
Feedback I left in https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/320/

We should separate these interfaces before deployment. Backlog item.

Reply:
Addressed the issue in common here https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/320 and via this commit here : https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/475/commits/2cb693a9eff22eeaefd9f8a2b119e0e691ef3960